### ISSUE TITLE
Localize Subscription Details

### ISSUE THREAD

**Issue by tiagonmas** [banco-alimentar/alimentestaideia.pt#631]
Not localized:

-  /Identity/Account/Manage/Subscriptions/Details
- /Identity/Account/Manage/Subscriptions/Delete

eg:
![image](https://user-images.githubusercontent.com/42462/149343722-6aa49672-88db-4eee-9944-59ec41da385c.png)

**Comment by crnegule** [comment]
@tiagonmas  this based on how it looks is the same with https://github.com/banco-alimentar/alimentestaideia.pt/issues/524. 
If its that should we close one of them ?

**Comment by tiagonmas** [comment]
good catch. I'll close #524 and leave this one open and assigned to you. thanks!
